### PR TITLE
Fix ngOnChange

### PR DESCRIPTION
**Pull request by ishaiborovoy**

### CODE REVIEW

File: projects/ui-framework/src/lib/form-elements/lists/multi-select/multi-select.component.ts [hunk 1]
```diff
@@ -66,7 +66,9 @@ export class MultiSelectComponent extends BaseSelectPanelElement implements OnCh
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    this.triggerValue = this.getTriggerValue(this.value);
+    if (changes.value) {
+      this.triggerValue = this.getTriggerValue(this.value);
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/form-elements/lists/multi-select/multi-select.component.ts]: it's cleaner for sure, but i will be surprised if this solves it, did you try other lifecycle hooks?

Comment (ishaiborovoy) [reply projects/ui-framework/src/lib/form-elements/lists/multi-select/multi-select.component.ts]: I solved the endless loop in form-generator and how to set the value, see the pool request there in hibob-web-client

Comment (ishaiborovoy) [reply projects/ui-framework/src/lib/form-elements/lists/multi-select/multi-select.component.ts]: I solved the endless loop in form-generator and how to set the value, see the pool request there in hibob-web-client

Comment (ishaiborovoy) [reply projects/ui-framework/src/lib/form-elements/lists/multi-select/multi-select.component.ts]: This is just to ensure you update when value changed and not on every other change

**Pull request merged**